src/vs/base/browser/markdownRenderer.ts

```diff
@@ -100,6 +100,22 @@ export function renderMarkdown(markdown: IMarkdownString, options: MarkdownRende
 	const withInnerHTML = new Promise<void>(c => signalInnerHTML = c);
 
 	const renderer = new marked.Renderer();
+
+	const htmlParser = new DOMParser();
+	renderer.html = (rawHtml: string) => {
```

Comment:
This seems to break markdown text such as: `text <b>bold</b>` as it gets rendered as:

```html
<p>text <b></b>bold</p>
```

Instead, I think you should move this logic after the full md has been rendered but before sanitization:

https://github.com/microsoft/vscode/blob/d209aa2b02f6cc99f3c985b23c99eb2346f9dbdb/src/vs/base/browser/markdownRenderer.ts#L260-L261